Source/WebKit/_WebKit_SwiftUI/API/WebView.swift

```diff
@@ -24,6 +24,25 @@
 public import SwiftUI
 public import WebKit
 
+private enum StateWithURLOrWebPage: DynamicProperty {
+    case state(State<WebPage>, URL?)
+    case webPage(WebPage)
+
+    var webPage: WebPage {
+        switch self {
+        case .state(let state, _): state.wrappedValue
+        case .webPage(let webPage): webPage
```

Comment:
```suggestion
        case let .state(state, _): state.wrappedValue
        case let .webPage(webPage): webPage
```

this is a style nit, but it's usually preferred to have the let at the beginning for consistency